doc/user_manual/OutStreamSystem.tex

```diff
@@ -1714,3 +1714,53 @@ \subsubsection{SamplePlot}
   </OutStreams>
 </Simulations>
 \end{lstlisting}
+
+\subsubsection{OptPath}
+The \xmlString{OptPath} \xmlAttr{subType} shows the path taken during a \xmlNode{MultiRun} using
+a gradient-based \xmlNode{Optimizer} as the sampling strategy. It is generally used to show the progressive
```

Comment:
only for the gradient-based optimizers? What about the others?

Reply:
The heuristic optimizers do not provide the same meta variables as gradient descent, since they don't work the same.

It's possible this could be extended in the future to capture the outputs of heuristic optimizations, as well; alternatively, a plotting system designed to work well for those algorithms could be implemented.

Reply:
This works fine for GA. I just suggest having a different color for 'first' and 'final' and add them to the legend.

Reply:
done